apps/s_server.c

```diff
@@ -2737,8 +2758,24 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
              */
             if ((!async || !SSL_waiting_for_async(con))
                     && !SSL_is_init_finished(con)) {
+                /*
+                 * Count number of reads during init_ssl_connection.
+                 * It helps us to recognise configuration errors and errors
```

Comment:
What about _It helps us to distinguish configuration errors from errors caused by a client_?